ricecooker/classes/nodes.py

```diff
@@ -607,7 +607,8 @@ class VideoNode(ContentNode):
             files ([<File>]): list of file objects for node (optional)
     """
     kind = content_kinds.VIDEO
-    required_file_format = file_formats.MP4
```

Comment:
Although it makes almost no difference because the validate function that used to use this has been commented out for some reason, I think it might be a neater convention to allow `required_file_format` to be either a string or a tuple, so I would changes this instead to:
```
required_file_format = (file_formats.MP4, file_formats.WEBM)
```

When we reinstate the validate function, we should then allow it to handle both strings and tuples as this property.